tests/ctst/steps/azureArchive.ts

```diff
@@ -449,6 +452,12 @@ When('i run sorbetctl to retry failed restore for {string} location', async func
     }
 });
 
+When('i wait for {int} days', { timeout: 10 * 60 * 1000 }, async function (this: Zenko, days: number) {
+    const realTimeDay = days * 24 * 60 * 60 * 1000 /
+        (this.parameters.TimeProgressionFactor !== undefined ? this.parameters.TimeProgressionFactor : 1);
```

Comment:
can be simplified like the bash code, to protect against other values

```suggestion
        (this.parameters.TimeProgressionFactor > 1 ? this.parameters.TimeProgressionFactor : 1);
```